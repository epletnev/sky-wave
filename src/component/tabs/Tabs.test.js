import React from "react";
import TabsCategory from "./Tabs";
// import renderer from "react-test-renderer";

// test("should render TabsCategory component", () => {
//   const tree = renderer.create(<TabsCategory />).toJSON();
//   expect(tree).toMatchSnapshot;
// });

test("expect to render TabsCategory component", () => {
  expect(<TabsCategory />).toMatchSnapshot();
});
