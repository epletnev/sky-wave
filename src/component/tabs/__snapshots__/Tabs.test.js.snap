// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render TabsCategory component 1`] = `<TabsCategory />`;

exports[`expext to render TabsCategory component 1`] = `<TabsCategory />`;
